stdlib/LinearAlgebra/src/generic.jl

```diff
@@ -1112,6 +1112,30 @@ function (\)(A::AbstractMatrix, B::AbstractVecOrMat)
 end
 
 (\)(a::AbstractVector, b::AbstractArray) = pinv(a) * b
+
+"""
+    /(B, X)
+
+For input matrices/vectors `B` and `X` where the number of columns of `B` and `X` should be same, 
+the resultant matrice `A` such that `A*X ≈ B`. The size of the resultant matrice depends on the row size
+of `B` and row size of `X`.
```

Comment:
Typically `X` should refer to the unknown you're solving for. Also, it should refer to `\`:

```suggestion
    A / B

Matrix right-division: `A / B` is equivalent to `(A' \ B')'` where [`\`](@ref) is the left-division operator.
For square matrices, the result `X` is such that `A == X*B`.

See also: [`rdiv!`](@ref).
```